actions/class.Category.php

```
@@ -59,6 +59,28 @@ public function getExposedsByClass()
+     *  taoItems/Category/getInvalidExposedCategories?id=${itemUri}
+     * Response
+     *  { success : bool, data : {
+     *      propertyUri : { label, values: [invalidValue] }
+     *  } }
+     */
+    public function getInvalidExposedCategories()
+    {
+        $id = $this->getRequestParameter('id');
+
+        if (is_null($id) || empty($id)) {
+            return $this->returnBadParameter('The item URI is required');
```

PHPStan (2.2.8)

[error] Result of method taoItems_actions_Category::returnBadParameter() (void) is used.


(method.void)